tests/check/test-basic.sh

```diff
@@ -45,6 +45,9 @@ ostree admin --sysroot=sysroot deploy --karg=root=LABEL=MOO --karg=quiet --os=te
 assert_status_jq '.deployments[0].version == "1.0.10"'
 echo "ok status shows right version"
 
+rpm-ostree reload
+echo "ok reload"
+
```

Comment:
(Well I didn't do the before/after since I'm not sure we should hardcode right now that we *don't* automatically pick up origin changes)